ruby/project_oop.md

```diff
@@ -190,7 +190,7 @@ Build a tic-tac-toe game on the command line where two human players can play ag
 * [Roland Studer's solution (with slow learning AI)](https://github.com/RolandStuder/odin_project_solutions/tree/master/oop/tic-tac-toe)
 * [Iceron's solution](https://github.com/iceron/TOP/blob/master/Ruby/oop/tic_tac_toe/tic_tac_toe.rb)
 * [Anistor86's solution](https://github.com/anistor86/tic_tac_toe)
-
+* [HuyAnh's solution](https://github.com/huyanh10tin/tictactoe)
```

Comment:
Can you add a new line below this please?

Reply:
ok